src/pwnedPasswordRange.ts

```diff
@@ -1,18 +1,14 @@
 import { fetchFromApi } from './api/pwnedpasswords';
 
 export interface PwnedPasswordSuffix {
```

Comment:
We should probably rename `PwnedPasswordSuffix` to `PwnedPasswordSuffixes` since it used to represent a single suffix but now it contains all matching suffixes.